pkg/ui/templates/query_menu.html

```diff
@@ -9,7 +9,7 @@
           <ul class="navbar-nav">
             <li class="nav-item"><a class="nav-link" href="{{ pathPrefix }}/graph">Graph</a></li>
             <li class="nav-item"><a class="nav-link" href="{{ pathPrefix }}/stores">Stores</a></li>
-	    <li class="nav-item dropdown">
+	        <li class="nav-item dropdown">
```

Comment:
I think this indentation is unneeded

Reply:
Thanks! I just format this page.

Reply:
thanks (: looks like we have a bit of a mix of 2sp and 4sp in those templates. I'll make a followup to standardize it.

Reply:
:+1: